moveit_ros/planning/rdf_loader/include/moveit/rdf_loader/synchronized_string_parameter.h

```diff
@@ -80,5 +84,7 @@ class SynchronizedStringParameter
   rclcpp::Publisher<std_msgs::msg::String>::SharedPtr string_publisher_;
 
   std::string content_;
+  boost::lockfree::spsc_queue<std::string> queue_;
```

Comment:
I would vote for a solution that doesn't require Boost since we're planning to get rid of it as a dependency

Reply:
There is no non-boost replacement for the spsc queue.  The plan to remove boost is to remove the parts of boost that have been migrated into the stl.  The spsc_queue is not one of those things.